server/services/group.js

```diff
@@ -89,9 +98,42 @@ function addAdmin(groupName, userId) {
     .then(validatedUserId => Group.addAdmin(groupName, validatedUserId));
 }
 
+function scheduleEPRotation(group) {
+  const rotationInterval = group.escalationPolicy.rotationIntervalInDays;
+  const nextRotateDate = buildRotateDate(group.lastRotated, rotationInterval);
+  return new Promise((resolve) => {
```

Comment:
If you need a Promise in the return, you can just do `return Promise.resolve(...)` and include the data you need to be wrapped in a promise.